rotkehlchen/tests/unit/decoders/test_rainbow.py

```diff
@@ -217,3 +217,318 @@ def test_rainbow_swap_token_to_token(ethereum_inquirer, ethereum_accounts):
     )]
 
     assert expected_events == events
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0xD918F8179e915e523c0d14B98dDD45aD6AE82076']])
+def test_rainbow_swap_on_arbitrum_one(arbitrum_one_inquirer, arbitrum_one_accounts):
+    tx_hash = deserialize_evm_tx_hash('0xeb91cb1f850ebf1ac028b9c23c2445e8d050df6eca58f9aff6a2f77e5d156fcc')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=arbitrum_one_inquirer, tx_hash=tx_hash)  # noqa: E501
+    gas_fees, swap_amount, received_amount, fee_amount, timestamp, user_address = '0.0000046705', '0.0305382', '57.839494', '0.0002618', TimestampMS(1742300433000), arbitrum_one_accounts[0]  # noqa: E501
+
+    expected_events = [EvmEvent(
```

Comment:
```suggestion
    assert events == [EvmEvent(
```
Don't use blank lines where the code is just as readable without them there.

Also you can do the assert right here at the beginning instead of using a `expected_events` variable.

Same comment for all the tests here.